core/drivers/scmi-msg/common.h

```diff
@@ -104,6 +104,12 @@ scmi_msg_handler_t scmi_msg_get_base_handler(struct scmi_msg *msg);
  * Return a function handler for the message or NULL
  */
 scmi_msg_handler_t scmi_msg_get_clock_handler(struct scmi_msg *msg);
+#else
+static inline scmi_msg_handler_t scmi_msg_get_clock_handler(
+		struct scmi_msg *msg __unused)
```

Comment:
checkpatch is not happy here:
```
CHECK: Lines should not end with a '('
#24: FILE: core/drivers/scmi-msg/common.h:108:
+static inline scmi_msg_handler_t scmi_msg_get_clock_handler(
```
Prefer to line break after `inline`?
```
static inline
scmi_msg_handler_t scmi_msg_get_clock_handler(struct scmi_msg *msg __unused)
```

Reply:
> Prefer to line break after `inline`?

Yes, fine with me.